sdk/eventhub/Azure.Messaging.EventHubs/src/ReceiverOptions.cs

```diff
@@ -16,14 +16,20 @@ namespace Azure.Messaging.EventHubs
     public class ReceiverOptions
     {
         /// <summary>The name of the default consumer group in the Event Hubs service.</summary>
-        public const string DefaultConsumerGroupName = "$Default";
+        public const string DefaultConsumerGroup = "$Default";
 
         /// <summary>The minimum value allowed for the prefetch count of the receiver.</summary>
         protected const int MinimumPrefetchCount = 10;
 
         /// <summary>The maximum length, in characters, for the identifier assigned to a receiver.</summary>
         protected const int MaximumIdentifierLength = 64;
 
+        /// <summary>The retry policy to apply to operations.</summary>
+        protected Retry _retry = Retry.Default;
```

Comment:
I don't think we use protected fields + public APIs shouldn't have `_` prefix.

Reply:
hrmm... good catch.   Not sure why those are `protected`.   Likely overlooked in a design change.   Will adjust for the next PR.